higher_health/forms.py

```diff
@@ -88,22 +88,24 @@ class HealthCheckQuestionnaire(forms.Form):
         label="", choices=DESTINATION_CHOICES, widget=forms.RadioSelect
     )
     facility_destination_province = forms.ChoiceField(
-        label="Please select a province", choices=[("", "--------")] + PROVINCE_CHOICES
+        label="Please select a province:", choices=[("", "--------")] + PROVINCE_CHOICES
     )
     facility_destination_university = forms.ModelChoiceField(
-        label="Please select an institution",
+        label="Please select an institution:",
         queryset=models.University.objects.all(),
         required=False,
     )
     facility_destination_university_other = forms.CharField(
-        label="Other", required=False
+        label="Enter a name of an institution:", required=False
```

Comment:
Shouldn't this be "Enter *the* name of *the* institution:"? (similarly for the campus question)
Please check with SxD on the wording